x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js

```diff
@@ -1392,6 +1392,63 @@ var sysmon = (function () {
         .Add(removeEmptyEventData)
         .Build();
 
+    // Event ID 23 - FileDelete (A file delete was detected).
+    var event23 = new processor.Chain()
+        .Add(parseUtcTime)
+        .AddFields({
+            fields: {
+                "event.category": ["file"], // pipes are files
+                "event.type": ["deletion"],
+            },
+        })
+        .Convert({
+            fields: [
+                {
+                    from: "winlog.event_data.UtcTime",
+                    to: "@timestamp",
+                },
+                {
+                    from: "winlog.event_data.ProcessGuid",
+                    to: "process.entity_id",
+                },
+                {
+                    from: "winlog.event_data.ProcessId",
+                    to: "process.pid",
+                    type: "long",
+                },
+                {
+                    from: "winlog.event_data.RuleName",
+                    to: "rule.name",
+                },
+                {
+                    from: "winlog.event_data.TargetFilename",
+                    to: "file.name",
+                },
+                {
+                    from: "winlog.event_data.Image",
+                    to: "process.executable",
+                },
+                {
+                    from: "winlog.event_data.Archived",
```

Comment:
The `process` namespace is "managed" by [ECS](https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html). If `archived` and `is_executable` are not part of ECS then we should not add them to `process`. 

Either leave them as-is or copy them over to a `sysmon` namespace like `sysmon.is_archived`. Then add an entry to this file for the field. https://github.com/elastic/beats/blob/master/x-pack/winlogbeat/module/sysmon/_meta/fields.yml The benefit of copying them is that we can set the correct data type. Leaving them in `winlog.event_data` means they will be a `keyword` in Elasticsearch rather than boolean.

Reply:
Added them to `sysmon.file.archived` and `sysmon.file.is_executable` 👍